spec/requests/api/v1/users/update_spec.rb

```diff
@@ -1,13 +1,22 @@
+require 'rails_helper'
```

Comment:
are we sure this is necessary?

Reply:
Odd, sometimes if I don't write it explicitly it won't work. In this particular case, it raises _undefined method 'create'_ when creating user.

Reply:
you are right 👍  this project doesn't have the `.rspec` file.

I recommend you add it, it's just a file at the root of the project named `.rspec` that should have
```
--require rails_helper
```
as it content.

That way rspec always loads that file before running specs 😉

Reply:
Excellent! 🚀